src/setupTests.js

```diff
@@ -3,3 +3,8 @@
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
 import '@testing-library/jest-dom/extend-expect';
+import '@testing-library/jest-dom/extend-expect';
+
```

Comment:
Don't forget to `npm i jest-enzyme` 

and then import it here: 

`import 'jest-enzyme';`